src/solvers/dg_tree/dg.jl

```diff
@@ -259,26 +259,26 @@ include("l2projection.jl")
 include("basis_lobatto_legendre.jl")
 
 """
-    DGSEM([RealT=Float64,] polydeg::Integer,
+    DGSEM(polydeg::Integer,
```

Comment:
Keep this one, please

Reply:
Oops - sorry. Will revert these changes.